state_machine/applications/flight/Tasks/beacon_task.py

```diff
@@ -6,19 +6,28 @@
 from pycubed import cubesat
 from state_machine import state_machine
 import struct
+import time
 
 class task(Task):
     name = 'beacon'
     color = 'teal'
+    fileindex = 1
 
     async def main_task(self):
         """
         Pushes a beacon packet onto the transmission queue.
         """
+        currTime = time.time()
+        TIMEINTERVAL = 1000
+        print(currTime)
+        file = open(f"./logfiles/log{int(currTime // TIMEINTERVAL)}.txt", "ab+")
```

Comment:
We'd probably like to do `f".sd/logfiles/log{int(currTime // TIMEINTERVAL)}.txt"`, please handle the case where the logfiles folder doesn't exist.